### PR TITLE
stolostron/backlog#24460 Add pull secret 2.5

### PR DESCRIPTION
**Pull request by cadawson**

### CODE REVIEW

File: clusters/import_gui.adoc [hunk 1]
```diff
@@ -17,13 +17,32 @@ Follow this procedure to import from the console. You need your terminal for aut
...
+If you need to create a new one, complete the following steps:
++
+. Download your Kubernetes pull secret from https://cloud.redhat.com/[cloud.redhat.com]. 
+
+. Add the pull secret to the namespace of your hub cluster. 
+
+. Run the following command to create a new secret in the `open-cluster-management` namespace: 
++
+----
+oc create secret generic pull-secret -n <open-cluster-management> --from-file=.dockerconfigjson=../../pull-secret --type=kubernetes.io/dockerconfigjson
```
Comment (skeeey) [inline clusters/import_gui.adoc]: the namespace `open-cluster-management` is the default hub namespace, and we may be better use a variable for `from-flie`, we may need to update the command to

```
oc create secret generic pull-secret -n <open-cluster-management> --from-file=.dockerconfigjson=<your-pull-secret-file-path> --type=kubernetes.io/dockerconfigjson
```

File: clusters/import_gui.adoc [hunk 2]
```diff
@@ -17,13 +17,32 @@ Follow this procedure to import from the console. You need your terminal for aut
...
+----
++
+Replace `open-cluster-management` with the name of the namespace of your hub cluster.
```
Comment (skeeey) [inline clusters/import_gui.adoc]: may be add some note to clear,  by default, the hub namespace is open-cluster-management

and replace the `your-pull-secret-file-path` to your pull secret file path

Comment (skeeey) [reply clusters/import_gui.adoc]: or use `-n <hub-namespace>` in the command,

Replace `hub-namepspace` with the name of the namespace of your hub cluster, by default, the the hub namespace is `open-cluster-management`

**Comment by cadawson** [comment]
Updated the content to address your comments. Thanks @skeeey!

**Review by skeeey — APPROVED** [review]

**Comment by cadawson** [comment]
This was peer reviewed in 2.6. Backported here.

**Pull request merged**